lib/bandit/http1/handler.ex

```diff
@@ -197,4 +197,31 @@ defmodule Bandit.HTTP1.Handler do
 
   def handle_info({:EXIT, _pid, :normal}, {socket, state}),
     do: {:noreply, {socket, state}, socket.read_timeout}
+
+  def handle_info(msg, state) do
+    # Copied verbatim from lib/elixir/lib/gen_server.ex
+    proc =
+      case Process.info(self(), :registered_name) do
+        {_, []} -> self()
+        {_, name} -> name
+      end
```

Comment:
Do you allow the process to be registered? If not you can skip this, otherwise it looks good to me :)

Reply:
Yep, handler processes themselves are able to accept any valid gen_server option via the `genserver_options` flag.